<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>A1Pattern</class>
 <widget class="QMainWindow" name="A1Pattern">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1171</width>
    <height>755</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>A1Pattern</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1151</width>
      <height>701</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>720</y>
      <width>241</width>
      <height>29</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Quick Construction</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Procedural Construction</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Clear Screen</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelTLp1">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>720</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Top-Left Corner</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBRp2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>720</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Bottom-right corner</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelY1">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>720</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Y1</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelX1">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>720</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>X1</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelX2">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>720</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>X2</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelY2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>720</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Y2</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>710</y>
      <width>16</width>
      <height>51</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>710</y>
      <width>20</width>
      <height>51</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="QDrawBresPB">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>720</y>
      <width>106</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Draw</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ddaPB">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>720</y>
      <width>51</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>DDA</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditX1">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>720</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditY1">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>720</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditX2">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>720</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditY2">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>720</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhDigitsOnly</set>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
